astroplan/observer.py

```diff
@@ -1958,3 +1958,27 @@ def tonight(self, time=None, horizon=0 * u.degree, obswl=None):
         end_time = self.sun_rise_time(start_time, which='next', horizon=horizon)
 
         return start_time, end_time
+
+    @property
```

Comment:
Properties for a class usually go after the `__init__` class, which is how the [`Target`](https://github.com/astropy/astroplan/blob/0a6b5eaeb9e474071e74906a6922a1bbe9f627f7/astroplan/target.py#L50) class is set up.

Reply:
Moved